onnxruntime/core/providers/qnn/builder/qnn_node_group/dq_q_fusion.cc

```diff
@@ -0,0 +1,179 @@
+#include "core/providers/qnn/builder/qnn_node_group/dq_q_fusion.h"
```

Comment:
NOTE: This file contains existing code that was refactored and moved to a separate file. It fuses DQ -> Q into a QNN Convert op.